content/docs/dvclive/api-reference/live/log_plot.md

```diff
@@ -0,0 +1,219 @@
+# Live.log_plot()
+
+Generates a
+[scikit learn plot](https://scikit-learn.org/stable/visualizations.html) and
+saves the data in `{path}/plots/{name}.json`.
+
+```py
+def log_plot(self, name: str, labels, predictions, **kwargs):
+```
+
+#### Usage:
+
+```py
+from dvclive import Live
+
+live = Live()
+
+y_true = [0, 0, 1, 1]
+y_pred = [1, 0, 1, 0]
+y_score = [0.1, 0.4, 0.35, 0.8]]
+live.log_plot("roc", y_true, y_score)
+live.log_plot("confusion_matrix", y_true, y_pred)
+```
+
+## Description
+
+Uses `name` to determine which plot should be generated. See
+[supported plots](#supported-plots).
+
+💡 The generated `{path}/plots/{name}.json` can be visualized with `dvc plots`.
+
+### Step updates
+
+`Live.log_plot()` can be currently only used when `step` is `None`.
```

Comment:
Is ther another reference where this is explained we can link to? Or expand here on what this means (step-less scenarios)